api/src/main/java/org/geysermc/geyser/api/GeyserApi.java

```diff
@@ -133,6 +134,44 @@ public interface GeyserApi extends GeyserApiBase {
     @NonNull
     PlatformType platformType();
 
+    /**
+     * Gets the version of Java Minecraft that is supported.
+     *
+     * @return the version of Java Minecraft that is supported
+     */
+    @NonNull
+    String getJavaMinecraftVersion();
+
+    /**
+     * Gets the version of the Java Minecraft protocol that is supported.
+     *
+     * @return the version of the Java Minecraft protocol that is supported
+     */
+    int getJavaProtocolVersion();
+
+    /**
+     * Gets the version of Bedrock Minecraft that is supported.
+     *
+     * @return the version of Bedrock Minecraft that is supported
+     */
+    @NonNull
+    String getBedrockMinecraftVersion();
+
+    /**
+     * Gets the version of the Bedrock Minecraft protocol that is supported.
+     *
+     * @return the version of the Bedrock Minecraft protocol that is supported
+     */
+    int getBedrockProtocolVersion();
```

Comment:
How would this work with Geyser supporting multiple Bedrock protocol versions?

Reply:
I mean currently its just the default protocol

This simplification doesn't seem ideal

Reply:
Could be a list of protocols.. but then the matching of version -> protocol number would be unclear

hence, version class with individual fields could make sense :p

Reply:
i suggest interface + record

if we want to expose protocol specific information in the future it can be done there